### PR TITLE
MAUI-849605 Polar Chart UG for Over View and Getting Started page in .NET MAUI 

### PR DESCRIPTION
**Pull request by snehakPV**
**Feature Description**

Updated UG for Polar Chart (new control).

**Analysis and Design**

Referred the Xamarin, WinUI and WPF UG
Xamarin - https://help.syncfusion.com/xamarin/charts/charttypes#polar-chart
WinUI - https://help.syncfusion.com/winui/polar-chart/overview
WPF - https://help.syncfusion.com/wpf/charts/seriestypes/radarandpolar

**Solution Description**

This UG document helps to guide the user to understand the feature.

## Output ##
MAUI_polar_chart

### CODE REVIEW

**Review by Saravanan-Madhesh — APPROVED** [review]
Changes fine.

**Comment by SaiyathAliFathima** [comment]
@Saravanan-Madhesh  , the CI issue is resolved, can you please merge this PR

File: MAUI/Polar-Charts/Getting-Started.md [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+        }   
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Register the handler
+
+Syncfusion.Maui.Core NuGet package is a dependent package for all Syncfusion controls in .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
```
Comment (M-A32) [inline MAUI/Polar-Charts/Getting-Started.md:56]: The Syncfusion.Maui.Core NuGet package is a dependent package for all Syncfusion controls in .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.

File: MAUI/Polar-Charts/Getting-Started.md [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+                new PlantData { Direction = "NorthEast", Tree = 85}
+            }; 
+        }
+    }
+
+{% endhighlight %} 
+
+{% endtabs %} 
+
+Create a `ViewModel` instance and set it as the chart's `BindingContext`. This enables property binding from `ViewModel` class.
```
Comment (M-A32) [inline MAUI/Polar-Charts/Getting-Started.md:138]: Create a `ViewModel` instance and set it as the chart's `BindingContext`. This enables property binding from the `ViewModel` class.

File: MAUI/Polar-Charts/Getting-Started.md [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+                               YBindingPath="Tree" />
+    </chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+    SfPolarChart chart = new SfPolarChart();
+
+    // Initializing primary axis
```
Comment (M-A32) [inline MAUI/Polar-Charts/Getting-Started.md:230]: // Initializing primary axis.

File: MAUI/Polar-Charts/Getting-Started.md [hunk 4]
```diff
@@ -0,0 +1,497 @@
...
+    CategoryAxis primaryAxis = new CategoryAxis();
+    chart.PrimaryAxis = primaryAxis;
+
+    //Initializing secondary Axis
```
Comment (M-A32) [inline MAUI/Polar-Charts/Getting-Started.md:234]: //Initializing secondary Axis.

File: MAUI/Polar-Charts/Getting-Started.md [hunk 5]
```diff
@@ -0,0 +1,497 @@
...
+    NumericalAxis secondaryAxis = new NumericalAxis();
+    chart.SecondaryAxis = secondaryAxis;
+
+    //Initialize the series
```
Comment (M-A32) [inline MAUI/Polar-Charts/Getting-Started.md:238]: //Initialize the series.

File: MAUI/Polar-Charts/Getting-Started.md [hunk 6]
```diff
@@ -0,0 +1,497 @@
...
+    PolarAreaSeries series = new PolarAreaSeries();
+    series.ItemsSource = (new ViewModel()).PlantDetails;
+    series.XBindingPath = "Direction";
+    series.YBindingPath = "Tree";
+
+    //Adding Series to the Chart Series Collection
```
Comment (M-A32) [inline MAUI/Polar-Charts/Getting-Started.md:244]: //Adding Series to the Chart Series Collection.

**Review by M-A32 — COMMENTED** [review]
I have completed the review.

**Pull request merged**